src/modules/reporting/actions/estimate-submit-market-contribute.js

```diff
@@ -0,0 +1,28 @@
+import { augur } from 'services/augurjs'
+import logError from 'utils/log-error'
+import noop from 'utils/noop'
+
+export const estimateSubmitMarketContribute = (marketId, amount, callback = logError) => (dispatch, getState) => {
+  const { loginAccount, marketsData } = getState()
+
+  const market = marketsData[marketId]
+  if (!market) return callback('Market not found')
+  const payoutNumerators = Array(market.numOutcomes).fill(0)
```

Comment:
Missing 'new' keyword. Like:
`const payoutNumerators = (new Array(market.numOutcomes)).fill(0)`

This bit isn't tested anyplace.

Reply:
Or should we use getPayoutNumerators selector?

Reply:
payout numerators don't matter for estimating gas, just need them to add up to numTicks.